body.tex

```diff
@@ -359,8 +363,11 @@ \section*{Random}
 range of alternative bit generators such as a cryptographic counter-based
 generators (\code{AESCtr}) and generators that expose hardware random number
 generators (\code{RDRAND})\cite{randomgen}.} Bit generators export a capsule
-containing pointers to functions that produce 64 bits, 32 bits, or a random
-double. They also expose CFFI and ctypes interfaces to the same three functions.
+containing pointers to functions that produce
+FIXME this doesn't make sense (it's the uniform stream of numbers that matters
+rather that "capsule of pointers", also what's "a random double"?):
+64 bits, 32 bits, or a random double.
```

Comment:
ah, it misses the word "integer" for 64/32-bit. I was sitting there thinking "a double is 64 -bits, this is duplicate":)